raysect/core/math/function/function1d/cmath.pyx

```diff
@@ -147,3 +147,29 @@ cdef class Atan4Q1D(Function1D):
 
     cdef double evaluate(self, double x) except? -1e999:
         return cmath.atan2(self._numerator.evaluate(x), self._denominator.evaluate(x))
+
+cdef class Sqrt1D(Function1D):
+    """
+    A Function1D class that implements the square root of the result of a Function1D object: atan(f())
+
+    :param Function1D function: A Function1D object.
+    """
+    def __init__(self, object function):
+
+        self._function = autowrap_function1d(function)
+
+    cdef double evaluate(self, double x) except? -1e999:
+        return cmath.sqrt(self._function.evaluate(x))
+
+cdef class Erf1D(Function1D):
+    """
+    A Function1D class that implements the error function of the result of a Function1D object: atan(f())
```

Comment:
Copy paste error! references atan()

Reply:
Fixed, 

I was monkey copying in attempts to preserve the style.... :-)